example/python/permissions/can_add_asset_qty.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import iroha
+import commons
+
+admin = commons.user('admin@test')
+alice = commons.user('alice@test')
+
+
+def genesis_tx():
+    test_permissions = iroha.RolePermissionSet([iroha.Role_kAddAssetQty])
+    tx = iroha.ModelTransactionBuilder() \
+        .createdTime(commons.now()) \
+        .creatorAccountId(admin['id']) \
+        .addPeer('0.0.0.0:50541', admin['key'].publicKey()) \
+        .createRole('admin_role', commons.all_permissions()) \
+        .createRole('test_role', test_permissions) \
+        .createDomain('test', 'test_role') \
+        .createAccount('admin', 'test', admin['key'].publicKey()) \
+        .createAccount('alice', 'test', alice['key'].publicKey()) \
+        .appendRole(admin['id'], 'admin_role') \
+        .createAsset('coin', 'test', 2) \
+        .build()
+    return iroha.ModelProtoTransaction(tx) \
+        .signAndAddSignature(admin['key']).finish()
+
+
+def add_asset_tx():
+    tx = iroha.ModelTransactionBuilder() \
+        .createdTime(commons.now()) \
+        .creatorAccountId(alice['id']) \
+        .addAssetQuantity(alice['id'], 'coin#test', '5000.99') \
+        .build()
+    return iroha.ModelProtoTransaction(tx) \
+        .signAndAddSignature(alice['key']).finish()
+
+
+print(admin['key'].privateKey().hex())
```

Comment:
Maybe move this framework-related code to separate class (and file) for reducing boilerplate?

Reply:
Discussed with Fedor in offline.
Let's keep this code for now. We expect the protocol of communication between BTF and external scripts to be changed, so then we can reduce the amount of such boilerplate code.